models/model.js

```diff
@@ -96,6 +96,12 @@ const { vote_increment } = newComment;
     
 
   ).then((result) => {
+    if (result.rows.length === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: `No user found for article: ${article_id}`,
```

Comment:
Careful, it's not the user we have not found, but the article that wasn't found. We don't expect a user in the request so there would be no way to know if the user exists or not